test/bst.t

```diff
@@ -245,3 +245,187 @@ Testing Environment
 	FOO=bar
 	ROOT=/
 	EXECUTABLE=/bin/true
+
+Verifying cgroup system config
+
+	$ if touch /sys/fs/cgroup/cgroup.procs         ; then exit 0; else exit 80; fi
+	$ if touch /sys/fs/cgroup/bst                  ; then exit 0; else exit 80; fi
+	$ if touch /sys/fs/cgroup/bst/cpu.max          ; then exit 0; else exit 80; fi
+	$ if touch /sys/fs/cgroup/bst/cpu.weight       ; then exit 0; else exit 80; fi
+	$ if touch /sys/fs/cgroup/bst/memory.min       ; then exit 0; else exit 80; fi
+	$ if touch /sys/fs/cgroup/bst/memory.low       ; then exit 0; else exit 80; fi
+	$ if touch /sys/fs/cgroup/bst/memory.high      ; then exit 0; else exit 80; fi
+	$ if touch /sys/fs/cgroup/bst/memory.max       ; then exit 0; else exit 80; fi
+	$ if touch /sys/fs/cgroup/bst/memory.swap.high ; then exit 0; else exit 80; fi
```

Comment:
This will likely make the tests never run, ever. A lot of systems don't configure swap, especially when they have ssds/nvmes, since swap tends to kill these disks faster.

It's important that the tests run at the very least in the CI process.

Reply:
I lowered the scope of the tests to what should run on most platforms.